src/main/scala/org/apache/spark/rdd/OrderedLeftJoinRDD.scala

```diff
@@ -0,0 +1,44 @@
+package org.apache.spark.rdd
+
+import org.apache.spark._
+import org.broadinstitute.hail.Utils._
+
+import scala.reflect.ClassTag
+
+class OrderedLeftJoinRDD[T, K, V1, V2](rdd1: OrderedRDD[T, K, V1], rdd2: OrderedRDD[T, K, V2])
```

Comment:
I think I would be inclined towards `left` and `right` instead of `rdd1` and `rdd2`, but don't change this unless you really want to.  It will involve a cascade of renaming.  You don't want `left` and `r1Partitions`.